machine/v1alpha1/types_openstack.go

```diff
@@ -366,3 +372,64 @@ type RootVolume struct {
 	// Deprecated: deviceType will be silently ignored. There is no replacement.
 	DeprecatedDeviceType string `json:"deviceType,omitempty"`
 }
+
+// blockDeviceStorage is the storage type of a block device to create and
+// contains additional storage options.
+type BlockDeviceStorage struct {
+	// type is the type of block device to create.
+	// This can be either "volume" or "local".
+	// +kubebuilder:validation:Enum="volume";"local"
```

Comment:
Kube enums should always be PascalCase
```suggestion
	// +kubebuilder:validation:Enum="Volume";"Local"
```